graylog2-web-interface/src/components/grok-patterns/BulkLoadPatternModal.jsx

```diff
@@ -57,6 +60,12 @@ class BulkLoadPatternModal extends React.Component {
       GrokPatternsStore.bulkImport(request, importStrategy).then(() => {
         UserNotification.success('Grok Patterns imported successfully', 'Success!');
         this._closeModal();
+
+        this.props.sendTelemetry(TELEMETRY_EVENT_TYPE.GROK_PATTERN.IMPORTED, {
```

Comment:
Is this really a user event? I would imagine the user event is the user clicking something that will trigger the import. The import itself (successful or not) is not related to what the user does and should be logged somewhere else.

Reply:
Yes, it is, in this new naming we send the user action on the entity 'Grok pattern imported' whenever possible, and when needed we also send the action (click etc.) like in the event definition.